pyoverkiz/utils.py

```diff
@@ -0,0 +1,64 @@
+"""Utils for Overkiz client"""
+from __future__ import annotations
+
+import re
+
+from pyoverkiz.types import JSON
+
+
+def obfuscate_id(id: str | None) -> str:
+    """Mask id"""
+    return re.sub(r"(SETUP)?\d+-", "****-", str(id))
+
+
+def obfuscate_email(email: str | None) -> str:
+    """Mask email"""
+    return re.sub(r"(.).*@.*(.\..*)", r"\1****@****\2", str(email))
+
+
+def mask(input: str) -> str:
```

Comment:
`mask` vs `obfuscate`, we have to choose.

Reply:
For now I just ported the names from our current `models.py`. However happy to rename and list as breaking change.